src/style.css

```
@@ -0,0 +1,39 @@
+body {
+  margin: 0;
+}
+
+.stars__star {
+  display: flex;
+  background: url(./images/star.svg) no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  justify-content: space-between;
```

No need for flex. When setting background image, you just want to operate on background position etc
```suggestion
  background: url(./images/star.svg) no-repeat;
  background-position: center; // combine it with other values above
  width: 16px; // variable
  height: 16px;  //variable
  margin-right: 4px;
```